boards/common-partitions.mk

```diff
@@ -0,0 +1,92 @@
+#
+# functions and definitions related to partitioning/formatting and using
+# diskimages
+#
+
+# The automatically created partition layout is always the same:
+# - part1, vfat, containing boot files and initrd
+# - part2, optional swap partition
```

Comment:
@hamishcoleman Is this a new feature that would allow creation of a swap partition? How will this be used/mounted as swap?

Reply:
It is not really a new feature - it is more making the setup from sun8i-v3s-licheepi-zero into a set of generically usable things.

I have not built anything to use/mount the swap yet, but I imagine that a build config option to format that partition and a systemd unit to search for a valid swap signature and use it would cover the bases